internal/contour/metrics_test.go

```diff
@@ -413,9 +413,7 @@ func TestHTTPProxyMetrics(t *testing.T) {
 			},
 			Valid:    map[metrics.Meta]int{},
 			Orphaned: map[metrics.Meta]int{},
-			Root: map[metrics.Meta]int{
-				{Namespace: "finance"}: 1,
-			},
+			Root:     map[metrics.Meta]int{},
```

Comment:
What's going on here, is the test wrong?

Reply:
It was, but I hadn't looked deeply enough into why; I needed to change the order of some things in `httpproxy_processor` for the information to flow out correctly.